client/src/components/App.jsx

```diff
@@ -3,7 +3,7 @@ import axios from 'axios';
 // eslint-disable-next-line import/extensions
```

Comment:
looks like all the eslint-disable-next-line appears couple times here and all commented out?